mcs/class/corlib/System/TimeZoneInfo.cs

```diff
@@ -119,6 +119,8 @@ static TimeZoneInfo CreateLocal ()
 			} catch {
 				try {
 					return FindSystemTimeZoneByFileName ("Local", Path.Combine (TimeZoneDirectory, "localtime"));	
+				} catch (TimeZoneNotFoundException) {
+					return Utc;
 				} catch {
 					return null;
 				}
```

Comment:
I don't know if we can just assume Utc if the file(s) are not found, @esdrubal any comments?

If assuming Utc is ok, then the catch that returns null should be removed imho, it's only used in the getter of `Local` property and immediately throws an exception there and we would only hit this case in extremely rare circumstances with this PR applied.